README.md

```diff
@@ -1,2 +1,45 @@
 # github-actions
 A collection of reusable and composite Github Actions
+
+## maven/build-scan-save
+A Composite action to save an unpublished Maven Build Scan®.
+
+The action uploads the Build Scan® (if present) as a workflow artifact with a random name (pattern: `scan-maven-<UUID>`).
+
+**Dependencies**:
+
+- [actions/upload-artifact](https://github.com/marketplace/actions/upload-a-build-artifact)
+
+**Event Trigger**:
+
+This composite action can be called from any workflow but the main use case is to save unpublished Build Scans® issued from workflows triggered on `pull_request` event
+
+**Action inputs**:
+
+N/A
+
+**Usage**:
+
+```yaml
+name: Check Pull Request
```

Comment:
I think we should rename the workflow (and Job) in the docs so that it's clear that this is an existing CI workflow (and only the "Save Build Scan" step is added).

Reply:
Sure